core/src/test/java/io/questdb/cutlass/line/tcp/LineTcpAuthConnectionContextTest.java

```diff
@@ -168,6 +168,28 @@ public void testGoodAuthentication() throws Exception {
         });
     }
 
+    @Test
+    public void testGoodAuthenticationWithExtraData() throws Exception {
+        runInAuthContext(() -> {
+            try {
+                boolean authSequenceCompleted = authenticate(AUTH_KEY_ID1, AUTH_PRIVATE_KEY1,
+                        "weather,location=us-midwest temperature=82 1465839830100400200\n");
+                Assert.assertTrue(authSequenceCompleted);
+            } catch (RuntimeException ex) {
+                // Expected that Java 8 does not have SHA256withECDSAinP1363
+                if (ex.getCause() instanceof NoSuchAlgorithmException && TestUtils.getJavaVersion() <= 8) {
```

Comment:
@glasstiger: on what Java you were getting this error? Was it in the CI? 
I tried it on Azul Zulu 8u332 and it does have the algorithm.

Reply:
that is a copy-paste from testGoodAuthentication().
i copied the test and modified it to send the ILP line together with the auth message.

Reply:
looks like @ideoma added this try-catch 2 months ago

Reply:
tagging @ideoma

Reply:
It didn't work on some Java 8 version a year ago. I don't remember which version exactly.
Free to remove it if you can run the test on latest Java 8.